### ISSUE TITLE
I cannot get the result in the paper on the voc 2007 dataset.

### ISSUE THREAD

**Issue by Horatio-Ng** [luiszeni/Boosted-OICR#8]
Hi, I didn't change configs but get a much lower result. I notice that:
Adaptive config changes:
    effective_batch_size: 2 --> 4
    NUM_GPUS:             1 --> 1
    IMS_PER_BATCH:        2 --> 1
Adjust BASE_LR linearly according to batch_size change:
    BASE_LR: 0.001 --> 0.0005
Adjust SOLVER.STEPS and SOLVER.MAX_ITER linearly based on effective_batch_size change:
    SOLVER.STEPS: [0, 35000] --> [0, 17500]
    SOLVER.MAX_ITER: 500000 --> 250000
Number of data loading threads: 4
********************************************************************************************************
Is it right? Does it mean my batch size is 4, which is 2 in the paper?

**Comment by luiszeni** [comment]
Hi, thanks for trying my code. Unfortunately, this is a known issue with my and PCL PyTorch code. 

You can enable the fixed seed in the training script and/or try to retrain the model. 

Anyway, if you enable the fixed seed, the results between different training sections using the same seed still differ.  I am investigating this nondeterminism present in my code, and I am not sure how to solve this problem yet :|.   

If any of you guys have experience with PyTorch and reproducibility, help would be very appreciable =)

**Comment by luiszeni** [comment]
Hi, I returned the code to an old version. Now the final mAP is really near to the results reported in the paper. I retrained the model with the same seed five times, and the final mAP oscillates between 49.0 and 49.9.

**Comment by Horatio-Ng** [comment]
> Hi, I returned the code to an old version. Now the final mAP is really near to the results reported in the paper. I retrained the model with the same seed five times, and the final mAP oscillates between 49.0 and 49.9.

thank you very much~
